planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -437,10 +437,24 @@ bool AvoidanceModule::canYieldManeuver(const AvoidancePlanningData & data) const
 {
   // transit yield maneuver only when the avoidance maneuver is not initiated.
   if (helper_.isShifted()) {
+    RCLCPP_DEBUG(getLogger(), "avoidance maneuver already initiated.");
     return false;
   }
 
+  // prevent sudden steering.
+  const auto registered_lines = path_shifter_.getShiftLines();
+  if (!registered_lines.empty()) {
+    const size_t idx = planner_data_->findEgoIndex(path_shifter_.getReferencePath().points);
+    const auto to_shift_start_point = calcSignedArcLength(
+      path_shifter_.getReferencePath().points, idx, registered_lines.front().start_idx);
+    if (to_shift_start_point < helper_.getMinimumPrepareDistance()) {
+      RCLCPP_DEBUG(getLogger(), "prevent sudden steering.");
```

Comment:
```suggestion
      RCLCPP_DEBUG(getLogger(), "prepare distance is not enough.");
```

Reply:
I think the suggested word is more accurate.
How do you think?

Reply:
Perfect :+1: